changelog-generator/content/changelogs/20230712_tasks_service-1-5-0.md

```diff
@@ -0,0 +1,27 @@
+---
+service: tasks-service
+version: 1.5.0
+date: 2023-07-12
+---
+
+
+
+**🎁 Features**
+* Added API Requests
+  * API Requests are a summary of the requests made to API Functions. They can be helpful to monitor and debug API Function calls.
+* Add functionality to consult logs of API Requests
+  * API Functions may now include logging statements during execution. These logs are saved as API Request Logs.
+  
+**⚒️ Improvements**
+* Increased performance of API Functions and Direct Function Execution
+* Improve the error message for tasks of which the output could not be determined
+* Updated the function runtimes
+  * The new runtimes are documented in the [public documentation](https://docs.extrahorizon.com/extrahorizon/services/automation/task-service/functions#runtime).
```

Comment:
(positive) I like the link to the docs here! We might want to think about doing this for features as well in the future, so people have a trigger to explore the new features in the documentation.